client/src/containers/index.tsx

```diff
@@ -0,0 +1,3 @@
+import AuthContainer from "./AuthContainer";
+
```

Comment:
Seems useless.. maybe we should use this file for a welcome screen in a later PR.

Reply:
Just using index.tsx as a centralized place for containers and components. We can definitely change this as we proceed.